modules/rosa-hcp-sts-creating-a-cluster-cli.adoc

```diff
@@ -28,35 +28,47 @@ When using the {product-title} (ROSA) CLI, `rosa`, to create a cluster, you can
 //REGION="<region>"
 //----
 
-. You can create your {hcp-title} cluster with one of the following commands.
+. Use the following command to create your {hcp-title} cluster:
 +
 [NOTE]
 ====
 When creating a {hcp-title} cluster, the default machine Classless Inter-Domain Routing (CIDR) is `10.0.0.0/16`. If this does not correspond to the CIDR range for your VPC subnets, add `--machine-cidr <address_block>` to the following commands. To learn more about the default CIDR ranges for {product-title}, see xref:../networking/cidr-range-definitions.adoc#cidr-range-definitions[CIDR range definitions].
 ====
 +
-** Create a cluster with a single, initial machine pool, publicly available API, and publicly available Ingress by running the following command:
-+
+--
 [source,terminal]
 ----
-$ rosa create cluster --cluster-name=<cluster_name> \
-    --sts --mode=auto --hosted-cp --operator-roles-prefix <operator-role-prefix> \
-    --oidc-config-id <ID-of-OIDC-configuration> --subnet-ids=<public-subnet-id>,<private-subnet-id>
+$ rosa create cluster --cluster-name=<cluster_name> \ <.>
+    --mode=auto --hosted-cp [--private] \ <.> 
+    --operator-roles-prefix <operator-role-prefix> \ <.>
+    --oidc-config-id <id-of-oidc-configuration> \ 
+    --subnet-ids=<public-subnet-id>,<private-subnet-id>
 ----
+<.> Specify the name of your cluster, ensuring that the cluster name consists of no more than 15 lowercase alphanumeric characters or '-', starts with a letter, and ends with an alphanumeric character.
+<.> Optional: The `--private` argument is used to create private {hcp-title} clusters. If you use this argument, ensure that you only use your private subnet ID for `--subnet-ids`.
+<.> By default, the cluster-specific Operator role names are prefixed with the cluster name and a random 4-digit hash. You can optionally specify a custom prefix to replace `<cluster_name>-<hash>` in the role names. The prefix is applied when you create the cluster-specific Operator IAM roles. For information about the prefix, see _About custom Operator IAM role prefixes_.
++
+[NOTE]
+====
+If you specified custom ARN paths when you created the associated account-wide roles, the custom path is automatically detected. The custom path is applied to the cluster-specific Operator roles when you create them in a later step.
+====
+--
+
+. If you set the environmental variables, create a cluster with a single, initial machine pool, using either a publicly or privately available API, and publicly or privately available Ingress by running the following commands:
 
-** Create a cluster with a single, initial machine pool, privately available API, and privately available Ingress by running the following command:
+.. For a private cluster, run:
```

Comment:
. If you set the environmental variables, create a cluster with a single, initial machine pool, a privately available API, and a privately available Ingress by running the following command: